OpenTabletDriver.UI.Library/Services/Windows/ShortcutAutoStartService.cs

```diff
@@ -0,0 +1,47 @@
+using System.Diagnostics;
+using WindowsShortcutFactory;
+
+namespace OpenTabletDriver.UI.Services.Windows;
+
+/// <summary>
+/// Creates a shortcut in the startup folder to start the application on login.
+/// </summary>
+public class ShortcutAutoStartService : IAutoStartService
+{
+    public bool AutoStartSupported => true;
+    public bool AutoStart => File.Exists(GetShortcutPath());
+
+    public string? BackendName => "Windows";
+
+    public bool TrySetAutoStart(bool autoStart)
+    {
+        if (!autoStart && AutoStart)
+        {
+            // Remove shortcut
+            File.Delete(GetShortcutPath());
+            return true;
+        }
+        else if (autoStart && !AutoStart)
+        {
+            // Create shortcut
+            using var shortcut = new WindowsShortcut
+            {
+                // TODO: Find a better way to get the executable path
+                Path = Path.Join(AppDomain.CurrentDomain.BaseDirectory, "OpenTabletDriver.UI.exe"),
+                Description = "Launch OpenTabletDriver",
+                ShowCommand = ProcessWindowStyle.Minimized
+            };
+
+            shortcut.Save(GetShortcutPath());
+            return true;
+        }
```

Comment:
add `else if (autoStart && AutoStart)` case to check if the already created shortcut matches current path?

Reply:
Will add shortcut target check to AutoStart instead, so launching from a different path will lead to AutoStart being false for that binary.